src/base/static/js/models/place-model.js

```diff
@@ -139,10 +139,19 @@ module.exports = Backbone.Model.extend({
     var attrs;
 
     if (method === "create" || method === "update") {
+      const keysToOmit = ["geometry"];
```

Comment:
addresses the "updating a place submitted by a logged in user" in https://github.com/jalMogo/mgmt/issues/32